stable/app/templates/deployment.yaml

```diff
@@ -21,6 +24,10 @@ spec:
         {{- end }}
       labels:
         {{- include "app.selectorLabels" . | nindent 8 }}
+        {{- include "app.labels" . | nindent 8 }}
```

Comment:
I think `app.labels` is not needed here since `selectorLabels` is included in `app.labels`.

Reply:
right. I think I get this wrong while editing last time. going to fix this